harmonica/forward/point_mass.py

```diff
@@ -189,6 +193,10 @@ def point_mass_gravity(
     # Convert to more convenient units
     if field == "g_z":
         result *= 1e5  # SI to mGal
+    if field == "g_northing":
```

Comment:
There's no need for creating one `if` statement for each component. The units conversion can be summarized as follows:
```python
if field in ("g_easting", "g_northing", "g_z"):
    result *= 1e5  # SI to mGal
```

Reply:
Perfect! I would suggest this modification in the future. Thank you